docs/guides/core-concepts/introduction-to-cypress.mdx

```diff
@@ -914,9 +914,9 @@ randomly fail. This would lead to flaky, inconsistent results.
 :::info
 
 While Cypress is built using Promises that come from
-[Bluebird](http://bluebirdjs.com/), these are not what we expose as commands and
-assertions on `cy`. If you'd like to learn more about handling asynchronous
-Cypress Commands please read our
+[Bluebird](http://bluebirdjs.com/), these are not what we expose as commands on
```

Comment:
This was there before, but it almost feels unnecessary to say "While Cypress is built using Promises that come from
[Bluebird](http://bluebirdjs.com/), these are not what we expose as commands on". We could keep it simple and just say "Commands on `cy` do not return Promises, although they are asynchronous".

Doesn't necessarily need to be updated here, tho